### ISSUE TITLE
rubick 004 queries can fail on newer versions

### ISSUE THREAD

**Issue by vikiival** [kodadot/nft-gallery#2960]
see the difference in casing :). It took me a couple of mins :)

004

```graphql
query MyQuery {
  nFTEntitiesConnection(orderBy: name_ASC)
}
```

005+

```graphql
query MyQuery {
  nftEntitiesConnection(orderBy: name_ASC)
}
```

**Comment by vikiival** [comment]
> are we using them somewhere?

rubick@004? yes for the landing page?

> are we using them somewhere?

004 and nft related queries? No

**Comment by yangwao** [comment]
Replace recent queries using rubick@004 where we are using them to the newest one
- on the landing page replace 004 to use rubick@006
- elsewhere where you find them

**Comment by yangwao** [comment]
Closed in #2969
